docs/modules/ROOT/pages/user-guide/dependency-management.adoc

```diff
@@ -2,6 +2,11 @@
 
 A specific Camel Quarkus release is supposed to work only with a specific Quarkus release.
 
+== Quarkus BOM 
+
+The core Quarkus dependencies are managed via `io.quarkus:quarkus-bom`. 
+Any additional dependencies for related Quarkus artifacts are not contained in this BOM. 
```

Comment:
Sorry, I cannot follow what you mean with "additional dependencies for related Quarkus artifacts". Could you please try to rephrase it?